CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+## [1.7.0] - 2019-04-19
```

Comment:
You should remove this. You are not releasing a not version, you are just saying that there is a new `Unreleased` feature.